internal/hcs/process.go

```diff
@@ -1,6 +1,7 @@
 package hcs
 
 import (
+	gcontext "context"
```

Comment:
Does anything collide with just using `context`?

Reply:
Yes

Reply:
I wonder if we should just fix those collisions.